ibis/expr/types/core.py

```diff
@@ -17,7 +17,7 @@
     IbisTypeError,
     TranslationError,
 )
-from ibis.common.grounds import Immutable
+from ibis.common.grounds import Immutable, console
```

Comment:
Could you move the `console` object to the `ibis.common.pretty` module?

Reply:
Yep.

Reply:
Done.